bioio/plugins.py

```diff
@@ -252,9 +252,17 @@ def dump_plugins() -> None:
         # Unpack dist
         dist = ep.dist
         if dist is not None:
-            print(f"  Author  : {dist.metadata['author']}")
+            if "author" in dist.metadata:
+                print(f"  Author  : {dist.metadata['author']}")
+            else:
+                print("  Author  : Not specified")
```

Comment:
I suggest trying to do it so that the string `"  Author  : "` is only in one print statement...  something more or less like
author = dist.metadata.get("author")
print(f"Author:{author if author is not None else "Not Specified"}")

This feels incrementally less error prone.

Reply:
resolved in [2c49cd8](https://github.com/bioio-devs/bioio/pull/59/commits/2c49cd8b4684ed8c28ed97c00caa294a453e554d)